health/warnings.go

```diff
@@ -0,0 +1,216 @@
+		}
+	},
+})
+
+// notInMapPollWarnable is a Warnable that warns the user that they cannot connect to the control server.
+var notInMapPollWarnable = Register(&Warnable{
+	Code:      "not-in-map-poll",
+	Title:     "Cannot connect to control server",
+	Severity:  SeverityMedium,
+	DependsOn: []*Warnable{NetworkStatusWarnable},
+	Text: func(args map[string]string) string {
+		return "Cannot connect to the control server (not in map poll). Check your Internet connection."
+	},
+})
+
+// noDERPHomeWarnable is a Warnable that warns the user that Tailscale doesn't have a home DERP.
+var noDERPHomeWarnable = Register(&Warnable{
+	Code:      "no-derp-home",
+	Title:     "No home relay server",
+	Severity:  SeverityHigh,
+	DependsOn: []*Warnable{NetworkStatusWarnable},
+	Text: func(args map[string]string) string {
+		return "Tailscale could not connect to any relay server. Check your Internet connection."
+	},
+})
+
+// noDERPConnectionWarnable is a Warnable that warns the user that Tailscale couldn't connect to a specific DERP server.
+var noDERPConnectionWarnable = Register(&Warnable{
+	Code:      "no-derp-connection",
+	Title:     "No relay server connection",
+	Severity:  SeverityHigh,
+	DependsOn: []*Warnable{NetworkStatusWarnable},
+	Text: func(args map[string]string) string {
+		return fmt.Sprintf("Tailscale could not connect to the relay server '%s'. The server might be temporarily unavailable, or your Internet connection might be down.", args["RegionID"])
+	},
+})
+
+// derpTimeoutWarnable is a Warnable that warns the user that Tailscale hasn't heard from the home DERP region for a while.
+var derpTimeoutWarnable = Register(&Warnable{
+	Code:      "derp-timed-out",
+	Title:     "Relay server timed out",
+	Severity:  SeverityMedium,
+	DependsOn: []*Warnable{NetworkStatusWarnable},
+	Text: func(args map[string]string) string {
+		return fmt.Sprintf("Tailscale hasn't heard from the home relay server (region %v) in %v. The server might be temporarily unavailable, or your Internet connection might be down.", args["RegionID"], args["Duration"])
+	},
+})
+
+// derpRegionErrorWarnable is a Warnable that warns the user that a DERP region is reporting an issue.
+var derpRegionErrorWarnable = Register(&Warnable{
+	Code:      "derp-region-error",
+	Title:     "Relay server error",
+	Severity:  SeverityMedium,
+	DependsOn: []*Warnable{NetworkStatusWarnable},
+	Text: func(args map[string]string) string {
+		return fmt.Sprintf("The relay server #%v is reporting an issue: %v", args["RegionID"], args["Error"])
+	},
+})
+
+// noUDP4BindWarnable is a Warnable that warns the user that Tailscale couldn't listen for incoming UDP connections.
+var noUDP4BindWarnable = Register(&Warnable{
+	Code:      "no-udp4-bind",
+	Title:     "Incoming connections may fail",
+	Severity:  SeverityHigh,
+	DependsOn: []*Warnable{NetworkStatusWarnable},
+	Text: func(args map[string]string) string {
+		return "Tailscale couldn't listen for incoming UDP connections."
+	},
```

Comment:
You have a bunch of these that make some text without using any args. Maybe a helper so we can write:

Text: health.StaticMesssage("foo")

where that's:

```go
func StaticMessage(s string) func(map[string]string) string {
  return func(map[string]string) string { return s }
}
```

Reply:
done